pkgs/os-specific/darwin/apple-source-releases/configd/default.nix

```diff
@@ -63,6 +64,8 @@ appleDerivation' stdenv {
     EOF
   '';
 
+  ${if headersOnly then "dontBuild" else null} = headersOnly;
```

Comment:
You are sure this evals when headersOnly is false?

Reply:
It definitely evals, although I never quiet understood how/if the trick works:
```
nix-repl> with import <nixpkgs> {}; stdenv.mkDerivation { name = "test"; }         
«derivation /nix/store/25nj2qsxjzrbnwjp924nfr0bfjksjf40-test.drv»

nix-repl> with import <nixpkgs> {}; stdenv.mkDerivation { name = "test"; null = 1; }
«derivation /nix/store/7zylvkwsk0gszax0x3i16ybfwsh3xrsr-test.drv»
```